bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/radiora/handler/DimmerHandler.java

```diff
@@ -34,43 +35,52 @@
  * @author Jeff Lauterbach - Initial Contribution
  *
  */
+@NonNullByDefault
 public class DimmerHandler extends LutronHandler {
 
     /**
      * Used to internally keep track of dimmer level. This helps us better respond
      * to external dimmer changes since RadioRA protocol does not send dimmer
      * levels in their messages.
      */
+    private @NonNullByDefault({}) DimmerConfig config;
     private AtomicInteger lastKnownIntensity = new AtomicInteger(100);
-
     private AtomicBoolean switchEnabled = new AtomicBoolean(false);
 
     public DimmerHandler(Thing thing) {
         super(thing);
     }
 
     @Override
-    public void handleCommand(ChannelUID channelUID, Command command) {
-        DimmerConfig config = getConfigAs(DimmerConfig.class);
+    public void initialize() {
+        config = getConfigAs(DimmerConfig.class);
+        super.initialize();
+    }
 
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        RS232Handler bridgeHandler = getRS232Handler();
```

Comment:
Could just bail out of this method at this point if bridgeHandler is null and avoid the duplicate null checks below

Reply:
Good point. I've changed that.